hospital/src/main/java/by/emel/anton/TerminalProgram.java

```diff
@@ -0,0 +1,234 @@
+import org.slf4j.LoggerFactory;
+
+public class TerminalProgram {
+
+    private static final String START_PROGRAM = "Hi, you are a doctor or patient? (d/p). If you are new user - create account(new)";
+    private static final String ANSWER_D = "d";
+    private static final String ANSWER_P = "p";
+    private static final String ANSWER_N = "n";
+    private static final String ANSWER_DOCTOR = "You are the doctor.";
+    private static final String ANSWER_PATIENT = "You are the patient";
+    private static final String CREATE_NEW_USER = "Create doctor(d) or patient(p)?";
+    private static final String ENTER_LOGIN = "Enter your login";
+    private static final String ENTER_PASSWORD = "Enter your password";
+    private static final String ENTER_LOG_PASSW = "Enter your login and password";
+    private static final String ENTER_NAME = "Enter your name";
+    private static final String ENTER_BIRTHDAY = "Enter you birthday yyyy-mm-dd";
+    private static final String HI = "Hi!";
+    private static final String PROCESSING_DOCTOR = "What do you want? See your patients (see)/Add new patient(add)/set therapy(set)?/exit(exit)";
+    private static final String ADD = "add";
+    private static final String SEE = "see";
+    private static final String SET = "set";
+    private static final String EXIT = "exit";
+    private static final String ENTER_PATIENT_ID = "Enter patient id";
+    private static final String ENTER_THERAPY_DESCR = "Enter therapy description";
+    private static final String ENTER_ENDDATE = "Enter end date (yyyy-mm-dd)";
+    private static final String PROCESSING_PATIENT = "What do you want? See your therapies (ts)/See you therapy(t)?/exit(exit)";
+    private static final String TS = "ts";
+    private static final String T = "t";
+    private static final String ENTER_ID_THERAPY = "Please, enter id therapy";
+    private static final String LOGGER_NAME = "LoggerForTerminal";
+
+
+    private final Logger logger = LoggerFactory.getLogger(LOGGER_NAME);
+    private final UserService userService;
+
+    public TerminalProgram(UserService userService) {
+        this.userService = userService;
+    }
+
+    public void startProgram() {
+
+        logger.info(START_PROGRAM);
+
+        try(Scanner scanner = new Scanner(System.in)) {
+
+            String answer = scanner.nextLine();
+
+            switch (answer) {
+                case ANSWER_D:
+                    logger.info(ANSWER_DOCTOR);
+                    enterDoctor(scanner);
+                    break;
+                case ANSWER_P:
+                    logger.info(ANSWER_PATIENT);
+                    enterPatient(scanner);
+                    break;
+                case ANSWER_N:
+                    createNewUser(scanner);
+                    break;
+            }
+            startProgram();
+        } catch (UserDAOException e) {
+            e.printStackTrace();
+        }
+
+    }
+
+    private void createNewUser(Scanner scanner)  {
```

Comment:
you can create 3 different classes with a common interface to perform all necessary logic inside.

Reply:
for new user, doctor, patient